undefined: setup

Getting:
<code>google/logger/logger.go:81:21: undefined: setup</code>
when running <code>go get github.com/google/logger</code>

go version go1.11.4 linux/amd64

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
 # logger #
-Logger is a simple cross platform Go logging library for Windows, Linux, and
+Logger is a simple cross platform Go logging library for Windows, Linux, FreeBSD, and
 macOS, it can log to the Windows event log, Linux/macOS syslog, and an io.Writer.
 
 This is not an official Google product.

--- a/logger_syslog.go
+++ b/logger_syslog.go
@@ -1,4 +1,4 @@
-// +build linux darwin
+// +build linux darwin freebsd
 
 /*
 Copyright 2016 Google Inc. All Rights Reserved.

